lib/13/libLLVM_h.jl

```diff
@@ -2077,6 +2077,7 @@ end
     LLVMModuleFlagBehaviorOverride = 3
     LLVMModuleFlagBehaviorAppend = 4
     LLVMModuleFlagBehaviorAppendUnique = 5
+    LLVMModuleFlagBehaviorMax = 6
```

Comment:
What is this for? It doesn't appear in the LLVM headers.

Reply:
Apple-compiled metal kernels use this flag behavior for various things. Some examples shown here:
```
!4 = !{i32 7, !"air.max_device_buffers", i32 31}
!5 = !{i32 7, !"air.max_constant_buffers", i32 31}
!6 = !{i32 7, !"air.max_threadgroup_buffers", i32 31}
!7 = !{i32 7, !"air.max_textures", i32 128}
!8 = !{i32 7, !"air.max_read_write_textures", i32 8}
!9 = !{i32 7, !"air.max_samplers", i32 16}
```

Reply:
You can just create a value for that without having to modify the headers:

```julia
julia> LLVM.API.LLVMModuleFlagBehavior(7)
UnknownMember::LLVMModuleFlagBehavior = 0x00000007
```